CovidTracker/Pages/Index.cshtml.cs

```diff
@@ -23,6 +24,22 @@ public void OnGet()
             using (var webClient = new WebClient())
             {
                 string jsonString = webClient.DownloadString("https://data.cdc.gov/resource/9mfq-cb36.json");
+
+                JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("StockSchema.json"));
+                IList<string> validationEvents = new List<string>();
+                JObject jsonObject = JObject.Parse(jsonString);
+                TrendingStock result = new TrendingStock();
+                if (jsonObject.IsValid(schema, out validationEvents))
+                {
+                    result = TrendingStock.FromJson(jsonString);
+                }
+                else
+                {
+                    //error handling
+                }
+
```

Comment:
This appears to be an incomplete feature.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality .
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.